package.json

```diff
@@ -76,7 +76,7 @@
   },
   "devDependencies": {
     "@formatjs/cli": "4.2.12",
-    "@redhat-cloud-services/frontend-components-config-utilities": "^1.5.20",
+    "@redhat-cloud-services/frontend-components-config-utilities": "^1.5.21",
```

Comment:
Would you please try updating the @redhat-cloud-services packages, again? Looks like there are latter versions now (e.g., ^1.5.23)